HISTORY.md

```diff
@@ -3,6 +3,8 @@
 
 ### mlpack 3.3.2
 ###### 2020-06-18
+  * Added N-step DQN to q_networks (#2461).
```

Comment:
I think this should go into a `mlpack ????` part, instead of this version

Reply:
Right! sorry about that.